src/bindings/c/ov/tests/ov_c_api_test.cpp

```diff
@@ -5,12 +5,15 @@
 #include <gtest/gtest.h>
 #include <stdio.h>
 #include <stdlib.h>
-#include <opencv2/opencv.hpp>
 #include <condition_variable>
 #include <mutex>
 #include "test_model_repo.hpp"
 #include <fstream>
 
+#ifdef BUILD_OPENCV_DEPENDENT_TESTS
```

Comment:
Let's remove dependency on opencv at all.
Based on comments below.

Reply:
yes, PR12187 does it.